packages/dd-trace/test/llmobs/plugins/ai/index.spec.js

```diff
@@ -372,15 +375,23 @@ describe('Plugin', () => {
           })
         })]
 
-        maxStepsArg = { maxSteps: 5 }
+        additionalOptions = { maxSteps: 5 }
+      }
+
+      if (semifies(openaiVersion, '>=2.0.50')) {
+        additionalOptions.providerOptions = {
+          openai: {
+            store: false
```

Comment:
If I understand this correct, it is set to false here to also increase the coverage for this particular case (having the store set to false and in other tests using the default, true). Is that correct? :)

Reply:
Thank you, that makes sense!